apps/aecore/include/aec_db.hrl

```diff
@@ -17,4 +17,4 @@
 -record(aec_tx_location        , {key, value}).
 -record(aec_tx_pool            , {key, value}).
 -record(aec_discovered_pof     , {key, value}).
-
+-record(aec_tx_gc              , {key, value}).
```

Comment:
Do I understand correctly this means that DB generated by new node cannot be used for old node? If so, could you please update https://github.com/aeternity/aeternity/blame/49ffb23ada36910dcda5bd87201cdf4072dcda7f/docs/release-notes/RELEASE-NOTES-2.4.0.md#L11

Reply:
As discussed offline, this is to be addressed.

Reply:
For clarity - new node needs to be able to start from DB of old node.

Re old node being able to start from new DB - not necessary provided documentation (release notes main doc) is fine tuned in order not to be misleading.